sdk/storage/storage-blob-changefeed/samples/javascript/sample.env

```diff
@@ -0,0 +1,3 @@
+# Used in most samples. Retrieve these values from a storage account in the Azure Portal.
+ACCOUNT_NAME=<storage-account-name>
+ACCOUNT_KEY=<storage-account-key>
```

Comment:
minor: blank line below

Reply:
I had the blank line locally. But they got removed probably by the formatting tool.
And I checked that most files in storage-blob in the master branch also does not have a blank line in the end.
https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/storage/storage-blob/samples/javascript/sample.env

So let's live with it.